.gitignore

```diff
@@ -21,3 +21,4 @@ testem.log
 .node_modules.ember-try/
 bower.json.ember-try
 package.json.ember-try
+.vscode/
```

Comment:
In general, editor specific ignores like this should be added to the [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)

Reply:
Will add.